### DISCUSSION TITLE
Discrepancy between versions 8.2 and 7.5

### DISCUSSION THREAD

**Discussion by gaurav-singh-au** [google/or-tools#2468] [CP-SAT questions] [answered]
Helllo @lperron and everyone, I believe there is a possible bug in the presolve as the attached model is infeasible in version 8.2 but feasible in 7.5! I think 7.5 is correct as at least 0 should be a feasible in this case. Interestingly, the model is infeasible from version 7.6 onwards so I believe it has been there for some time. Any help or advice you can offer will be appreciated. I am using python interface. 
[inf_cp_sat.txt](https://github.com/google/or-tools/files/6228002/inf_cp_sat.txt)

**Comment by lperron** [comment]
Thanks, we will have a look.

**Comment by lperron** [comment, marked as answer]
We have fixed this internally. I will try to push the fix later this week. Thanks for the report.

**Reply by gaurav-singh-au** [reply]
Nice to hear. Will look forward to the new version. Cheers.

**Reply by gaurav-singh-au** [reply]
@lperron, I trust you are well. Can you please advice if the fix for this bug has been pushed. Otherwise, if you can please email me the fix that will be great. Thanks.

**Comment by lperron** [comment]
Yes

Le lun. 12 avr. 2021 à 07:26, gaurav-singh-au ***@***.***> a
écrit :

**Comment by lperron** [comment]
It has been pushed last week.
